yarn.lock

```diff
@@ -1578,6 +1578,27 @@
     xcode "^3.0.1"
     xml2js "0.4.23"
 
+"@expo/config-plugins@~5.0.3":
```

Comment:
Avoid duplicating deps

Reply:
do you have any idea what caused this? I don't see anything related to that package in my code

Reply:
Run `yarn why @expo/config-plugins` to find which package is depending on the wrong version.

Reply:
Thanks! So... is it better to keep doctor at `@expo/config` 6.0.24 like the rest of the repo, or upgrade the repo to the latest?